pkgs/shells/oil/default.nix

```diff
@@ -19,7 +19,7 @@ stdenv.mkDerivation rec {
 
   strictDeps = true;
   buildInputs = lib.optional withReadline readline;
-  configureFlags = lib.optional withReadline "--with-readline";
+  configureFlags = lib.optionals withReadline [ "--with-readline" ];
```

Comment:
Do we have a written policy for this? As long as we don't, this is a bad idea (people have made changes in the opposite direction too).

Reply:
I've seen a couple people refer to https://discourse.nixos.org/t/lib-optional-considered-harmful/7081 lately, so I assumed there was consensus while there actually isn't.
I can revert if you wish.

Reply:
Commented my feelings there, thanks for the link :)

Reply:
Reverted.